cms/search/tests/test_wagtail_hooks.py

```diff
@@ -0,0 +1,96 @@
+from unittest.mock import patch
+
+import wagtail.coreutils
+from django.test import TestCase
+
+from cms.articles.tests.factories import ArticleSeriesPageFactory, StatisticalArticlePageFactory
+from cms.home.models import HomePage
+from cms.methodology.tests.factories import MethodologyPageFactory
+from cms.release_calendar.models import ReleaseCalendarIndex
+from cms.release_calendar.tests.factories import ReleaseCalendarPageFactory
+from cms.search.wagtail_hooks import page_published, page_unpublished
+from cms.standard_pages.tests.factories import IndexPageFactory, InformationPageFactory
+from cms.themes.tests.factories import ThemePageFactory
+from cms.topics.tests.factories import TopicPageFactory
+
+# Pages to exclude
+
+# Article series
+# Home page
+# Release calendar index
+# Theme page
+# Topic Page
+
+# Pages to include
+
+# Information Page: Topics: TESTED
+# Methodology Page: Topics: TESTED
+# Release Calendar Page: No topics:
+# Index Page: No topics
+# Statistical Article: No topics
+
+
+class WagtailHooksTests(TestCase):
+    @classmethod
+    def setUpTestData(cls):
+        cls.mock_request = wagtail.coreutils.get_dummy_request()
+
+        cls.excluded_factories = [
+            ArticleSeriesPageFactory,
+            HomePage,
+            ReleaseCalendarIndex,
+            ThemePageFactory,
+            TopicPageFactory,
+        ]
+
+        cls.included_factories = [
+            InformationPageFactory,
+            MethodologyPageFactory,
+            ReleaseCalendarPageFactory,
+            StatisticalArticlePageFactory,
+        ]
+
+        cls.index_page = IndexPageFactory(slug="custom-slug-1")
+        cls.included_factories.append(lambda: cls.index_page)
```

Comment:
considering you're initialising the pages anyway, why not store them as instances?

Reply:
Like this suggestion. Done!